LICENSE.md

```diff
@@ -1,2 +1,9 @@
 The DSP module is distributed under the MIT license.
 
+Copyright 2021 DSP.jl contributors
```

Comment:
The license file was initially introduced in 2012. Should this state the current year or the year of initial copyright?

Reply:
initial copyright is what matters for when it passes into the public domain, although it's common to include a range